modules/compiler/utils/source/replace_mux_math_decls_pass.cpp

```diff
@@ -51,15 +54,24 @@ PreservedAnalyses compiler::utils::ReplaceMuxMathDeclsPass::run(
         (0 == (DI.half_capabilities & device_floating_point_capabilities_full));
   }
 
-  const std::array<std::pair<StringRef, const bool>, 3> MuxMathDecls = {
-      std::make_pair(MuxBuiltins::isftz, flush_denorms_to_zero),
-      std::make_pair(MuxBuiltins ::usefast, UseFast),
-      std::make_pair(MuxBuiltins::isembeddedprofile, is_embedded_profile)};
+  // "Native" FMA means we can translate @llvm.fma without libcall support.
+  // TODO We want to handle this for x86, when targetting a sufficiently recent
+  // CPU, too, but this information is not in the target triple.
+  bool has_native_fma = TT.isARM() || TT.isRISCV();
```

Comment:
How about `AArch64`? I don't see any reference to it in the code or in any comments.

Reply:
Yep, should definitely add that, thanks.

Reply:
Done.